kitsune/sumo/static/sumo/js/form-wizard.js

```diff
@@ -46,10 +44,17 @@ export class FormWizard extends HTMLElement {
     super();
     let shadow = this.attachShadow({ mode: "open" });
 
+    let stylesheet = document.createElement("link");
+    stylesheet.setAttribute("rel", "stylesheet");
+    stylesheet.setAttribute(
+      "href",
+      this.getAttribute("static-url") + "form-wizard-styles.css"
```

Comment:
This is the part I think will be problematic in production, since the filename will be `form-wizard-styles.[some-hash].css`